packages/oracle-node/src/fetchers/api-fetcher/ApiFetcher.ts

```diff
@@ -0,0 +1,72 @@
+import axios, { AxiosResponse } from "axios";
+import jp from "jsonpath";
+import Decimal from "decimal.js";
+import {
+  MultiRequestFetcher,
+  RequestIdToResponse,
+} from "../MultiRequestFetcher";
+import { getLastPriceOrFail } from "../../db/local-db";
+
+interface ApiFetcherConfig {
+  url: string;
+  jsonpath: string;
+  pairedToken?: string;
+  isReverted?: boolean;
+}
+
+export class ApiFetcher extends MultiRequestFetcher {
+  constructor(
+    apiName: string,
+    private apiFetcherConfig: Record<string, ApiFetcherConfig>
+  ) {
+    super(`${apiName}-api-fetcher`);
+  }
+
+  override async makeRequest(
+    dataFeedId: string
+  ): Promise<AxiosResponse<unknown>> {
+    const { url } = this.apiFetcherConfig[dataFeedId];
+    return (await axios.get(url)).data as AxiosResponse<unknown>;
```

Comment:
Technically it is better to do it like this:
```suggestion
    return (await axios.get<unknown>(url)).data;
```
without explicit type assertions.